src/pages/installing/air-gap-installation.md

```diff
@@ -121,6 +127,7 @@ Log into the Red Hat OpenShift Container Platform cluster as a cluster administr
    ```
    cloudctl case launch \
        --case $HOME/offline/<case-file> \
+       --namespace openshift-marketplace \
```

Comment:
I haven't looked at the script, but what happens if a different namespace is used? Is it required that the openshift-marketplace namespace is used? I think when I ran this test, I used my acd namespace, such as "--namespace cag-test" for this step?